src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -63,6 +63,7 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
             BindToOption(DisplayInlineParameterNameHints, FeatureOnOffOptions.InlineParameterNameHints, LanguageNames.CSharp);
             BindToOption(GenerateXmlDocCommentsForTripleSlash, FeatureOnOffOptions.AutoXmlDocCommentGeneration, LanguageNames.CSharp);
             BindToOption(ShowRemarksInQuickInfo, QuickInfoOptions.ShowRemarksInQuickInfo, LanguageNames.CSharp);
+            BindToOption(InsertSlashSlashAtTheStartOfNewLinesWhenWritingSingleLineComments, SplitStringLiteralOptions.Enabled, LanguageNames.CSharp);
```

Comment:
💡 If the only options impacted by maintaining ordering are the three options moved to a new group, can we put those three bindings together?